### ISSUE TITLE
Pre-built binary install script does not work on MacOS Ventura

### ISSUE THREAD

**Issue by brombal** [casey/just#1532]
I noticed the pre-built binary install script does not work on Ventura (13.1, haven't tried 13.2 yet however). This is because, for some reason, the `--tlsv1.3` flag is not supported in the version that comes with Ventura:

```
$ curl --proto '=https' --tlsv1.3 -sSf https://just.systems/install.sh | bash -s -- --to [...]
curl: (4) A requested feature, protocol or option was not found built-in in this libcurl due to a build-time decision.
```

Replacing it with `--tlsv1.2` gets past this error, but the install.sh script itself also includes the flag, so it still fails.

The curl version included with Ventura is 7.85.0. If I use brew to install curl instead, it gives me 7.87.0, which works. However, the brew installation is keg-only, meaning it does not add it to the $PATH because replacing the built-in version can cause issues (this is brew's warning - not mine; but I agree with it).

I think the resolution is either:

1. Fall back to `--tlsv1.2` (I'm not sure what the security implications are of this).
2. Warn Ventura users about this issue, and recommend that they install curl using brew and then add it to their PATH, despite it being against brew's recommendation.

This issue may resolve itself with some future update to Ventura; and as there is a viable workaround, this issue may not be high priority. But hopefully it's at least helpful to anyone who Googles this issue.

TLDR:

```
$ brew install curl
$ PATH="/opt/homebrew/opt/curl/bin:$PATH"
$ curl --proto '=https' --tlsv1.3 -sSf https://just.systems/install.sh | bash -s -- --to DIR
```

**Comment by casey** [comment]
Thanks for the report!

I upgraded to 1.3 because it was new and shiny, and not for any particular pressing security reasons. You would think that after 30 years of using computers, I would have learned my lesson about upgrading to new and shiny things 😅

Reverted in #1536.

**Comment by brombal** [comment]
Definitely guilty of that myself, too :) thanks!
